Gemfile

```diff
@@ -19,4 +19,5 @@ group :test do
   gem 'rspec-rails', '~> 2.14'
   gem 'pry-rails',   '~> 0.3'
   gem 'generator_spec'
+  gem 'responders', '~> 2.0'
```

Comment:
thanks for the pr but this is not the place where to put the gem. This is for test env :) Try adding it after this line https://github.com/gottfrois/dashing-rails/blob/master/dashing.gemspec#L26 and run the `bundle` command to make sure it builds.